### PR TITLE
Feature/issue 384 52 revision

### PR DESCRIPTION
**Pull request by afsmythe**
This PR references the following open Issues:
https://github.com/votinginfoproject/vip-specification/issues/351
https://github.com/votinginfoproject/vip-specification/issues/162
https://github.com/votinginfoproject/vip-specification/issues/377
https://github.com/votinginfoproject/vip-specification/issues/367

This is a first pass at the 5.2 revision. Comments and suggestions are welcomed and encouraged.

### CODE REVIEW

File: docs/built_rst/csv/elements/polling_location.rst [hunk 1]
```diff
@@ -7,61 +7,73 @@ polling_location
...
++--------------------------------------+--------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
 
 .. code-block:: csv-table
    :linenos:
 
 
-    id,name,address_line,directions,hours,photo_uri,hours_open_id,is_drop_box,is_early_voting,latitude,longitude,latlng_source
-    poll001,ALBERMARLE HIGH SCHOOL","2775 Hydraulic Rd Charlottesville, VA 22901,Use back door,7am-8pm,www.picture.com,ho001,false,true,38.0754627,78.5014875,Google Maps
+    id,name,address_line,structured_location_name,structured_line_1,structured_city,structured_state,structured_zip,directions,hours,photo_uri,hours_open_id,is_drop_box,is_early_voting,latitude,longitude,latlng_source
+    poll001,ALBERMARLE HIGH SCHOOL","2775 Hydraulic Rd Charlottesville, VA 22901,ALBERMARLE HIGH SCHOOL,2775 Hydraulic Rd,Charlottesville,VA,22901,Use back door,7am-8pm,www.picture.com,ho001,false,true,38.0754627,78.5014875,Google Maps
```
Comment (jdmgoogle) [inline docs/built_rst/csv/elements/polling_location.rst]: This doesn't seem to be a valid example. It has both address_line and the structured address. Plus I'm not sure what the quotations are doing here.

File: docs/built_rst/csv/elements/street_segment.rst [hunk 2]
```diff
@@ -67,6 +67,14 @@ are equal.
 |                        |                           |              |              | **IncludesAllStreets** are true, this    |                                          |
 |                        |                           |              |              | value is ignored.                        |                                          |
 +------------------------+---------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| house_number_prefix    | ``xs:string``             | Optional     | Single       | Part of a street address. It may contain | If the field is not present of invalid,  |
```
Comment (jdmgoogle) [inline docs/built_rst/csv/elements/street_segment.rst]: I'd like to discuss these more before adding them.

File: docs/built_rst/tables/elements/street_segment.rst [hunk 3]
```diff
@@ -57,6 +57,14 @@
 |                      |                           |              |              | **IncludesAllStreets** are true, this    |                                          |
 |                      |                           |              |              | value is ignored.                        |                                          |
 +----------------------+---------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| HouseNumberPrefix    | ``xs:string``             | Optional     | Single       | Part of a street address. It may contain | If the field is not present of invalid,  |
```
Comment (jdmgoogle) [inline docs/built_rst/tables/elements/street_segment.rst]: Same here. We need guidance on what these mean before adding them. See my comments/requests on #162.

File: docs/yaml/elements/lat_lng.yaml [hunk 4]
```diff
@@ -3,23 +3,6 @@ csv-header-name: lat_long
 description: |-
   The latitude and longitude of a polling location in `WGS 84`_ format. Both
   latitude and longitude values are measured in decimal degrees.
-post: |-
```
Comment (jdmgoogle) [inline docs/yaml/elements/lat_lng.yaml]: Why was this block removed?

File: docs/yaml/elements/street_segment.yaml [hunk 5]
```diff
@@ -112,6 +112,22 @@ tags:
     **StartHouseNumber**, the implementation should ignore the element containing
     it.
   type: xs:integer
+- _name: HouseNumberPrefix
```
Comment (jdmgoogle) [inline docs/yaml/elements/street_segment.yaml]: See other comments about needing to resolve issues around #162 before adding this to the spec.

File: docs/yaml/elements/lat_lng.yaml (hunk 4 above)
Comment (afsmythe) [reply docs/yaml/elements/lat_lng.yaml]: This `PollingLocation` XML example post was relocated from the lat_long.yaml to the [polling_location.yaml](https://github.com/votinginfoproject/vip-specification/blob/ba3eaa2a0dd359ea663c317f21697a3dfd29292a/docs/yaml/elements/polling_location.yaml#L18) file.

File: docs/built_rst/csv/elements/polling_location.rst (hunk 1 above)
Comment (afsmythe) [reply docs/built_rst/csv/elements/polling_location.rst]: Will clean this up. It is a bad example.

**Pull request closed without merging**